code/game/objects/items/devices/uplinks.dm

```diff
@@ -211,5 +211,35 @@ var/list/world_uplinks = list()
 	hidden_uplink = new(src)
 	hidden_uplink.uses = 20
 
+/obj/item/device/radio/uplink/attackby(obj/item/I as obj, mob/user as mob)
+	if(istype(I, /obj/item/weapon/syndie_voucher))
+		RedeemVoucher(I, user)
+		return
 
-
+/obj/item/device/radio/uplink/proc/RedeemVoucher(obj/item/weapon/syndie_voucher/voucher, mob/redeemer)

```

Comment:
This isn't a good way to go about it since it only applies to the nuke op uplink intentionally